src/typescript/example.mainnet.env

```diff
@@ -46,6 +46,7 @@ NEXT_PUBLIC_DISCORD_METADATA_REQUEST_CHANNEL=""
 HASH_SEED="some random string that is not public"
 GEOBLOCKED='{"countries":[],"regions":[]}'
 NEXT_PUBLIC_CDN_URL=""
+RATE_LIMITING_ENABLED="false"
```

Comment:
- [ ] Per ECO-3122 explain how this requires the extra service from Vercel marketplace and is only useful in production